application/frontend/controllers/MfaController.php

```diff
@@ -179,6 +183,10 @@ protected function getRequestedMfa($id)
             );
         }
 
+        if ($mfa->type != Mfa::TYPE_WEBAUTHN) {
+            return $mfa;
+        }
```

Comment:
I don't think we can actually return yet for all non-WebAuthn MFA records, since the `loadData()` call below adds data for other types of MFA records as well. Was there a need to return here, or was it simply because the code below seemed unnecessary for non-WebAuthn MFA records (or for some other reason)?